en/config/safety.md

```diff
@@ -190,7 +190,7 @@ The failure action is controlled by [COM_POSCTL_NAVL](../advanced_config/paramet
 * `1`: Remote control *not* available.
   Switch to *Land mode* if a height estimate is available, otherwise enter flight termination.
 
-Fixed Wing vehicles additionally have a parameter ([NAV_GPSF_LT](../advanced_config/parameter_reference.md#NAV_GPSF_LT)) for defining how long they will loiter (circle) after losing position before attempting to land.
+Fixed-wing vehicles (and VTOLs in fixed-wing flight) additionally have a parameter ([FW_GPSF_LT](../advanced_config/parameter_reference.md#FW_GPSF_LT)) for defining how long they will loiter (circle) with a constant roll angle ([FW_GPSF_R](../advanced_config/parameter_reference.md#FW_GPSF_R)) at the current altitude after losing position efore attempting to land. If VTOLs have are configured to switch to hover for landing ([NAV_FORCE_VT](../advanced_config/parameter_reference.md#NAV_FORCE_VT)) then they will first transition and then descend.
```

Comment:
Excellent. FYI nothing wrong with what you wrote. I just wanted to fix the pre-existing typo.

```suggestion
Fixed-wing vehicles and VTOLs in fixed-wing flight additionally have a parameter ([FW_GPSF_LT](../advanced_config/parameter_reference.md#FW_GPSF_LT)) that defines how long they will loiter (circle with a constant roll angle ([FW_GPSF_R](../advanced_config/parameter_reference.md#FW_GPSF_R)) at the current altitude) after losing position before attempting to land.
If VTOLs have are configured to switch to hover for landing ([NAV_FORCE_VT](../advanced_config/parameter_reference.md#NAV_FORCE_VT)) then they will first transition and then descend.
```